src/components/elements/CommonHtmlContent.tsx

```diff
@@ -1,14 +1,20 @@
 import { Typography, TypographyProps } from '@mui/material';
+import DOMPurify from 'dompurify';
+import { useMemo } from 'react';
 
 interface Props extends TypographyProps {
   children: string;
 }
 const CommonHtmlContent: React.FC<Props> = ({ children, ref, ...props }) => {
+  const purifiedHtml = useMemo(() => {
```

Comment:
@ttoomey do you think it's a good idea to move this purification step inside of the `CommonHtmlContent` as I have done here? Or better to keep it outside, and have the components that want this call it separately as was the case before?

Reply:
That's a nice cleanup, thanks!!